### ISSUE TITLE
ExtPlane causing mouse cursor/button issues in Xplane 11 ?

### ISSUE THREAD

**Issue by BravoFoxMike** [vranki/ExtPlane#25]
Hi,

I'm trying to use the ExtPlane plugin with Xplane 11 (for the XPlanbeCDU app) but the mouse cursor turns weird (manipulators not always showing or slightly misplaced) and the right mouse button keeps getting stuck (or clicks double) randomly when the ExtPlane plugin is installed and running.

**Comment by vranki** [comment]
This sounds really weird, as ExtPlane doesn't do anything related to input or graphics. Needs testing.

**Comment by waynepiekarski** [comment]
@BravoFoxMike does the problem occur when XPlaneCDU is not running? I've seen a few reports of this kind of strange behavior, and it might be a memory corruption issue. But I'm curious if the problem happens when ExtPlane runs by itself, or if things only start to fail once a client like XPlaneCDU connects in and starts requesting datarefs.

**Comment by waynepiekarski** [comment]
@BravoFoxMike it looks like @vranki has created some new binaries for the ExtPlane plugin with all the newest fixes. It would be good for you to try these new versions, and see if the problem with the mouse cursor persists. It might be that my build was somehow not done properly (I used Cygwin) and causing a memory corruption issue, and it would be good to see if this is the cause. https://github.com/vranki/ExtPlane/releases/

**Comment by waynepiekarski** [comment]
@SHPEK1 which version of the ExtPlane plugin are you running? My version, or the latest builds from vranki?

**Comment by SHPEK1** [comment]
@waynepiekarski the latest one.

**Comment by waynepiekarski** [comment]
@SHPEK1 I'm not sure which one you mean. Can you paste in a link of the one you downloaded?

**Comment by SHPEK1** [comment]
@waynepiekarski I tried last release from vranki, still same problem

https://github.com/vranki/ExtPlane/releases/download/untagged-84f169c58f000e3b72d8/extplane.zip

**Comment by sum1els** [comment]
I am yet to experience or see this issue. I have seen many reports on the X-Plane 11 facebook group about a "cursor issue" unrelated to ExtPlane, so not sure if it's the same issue. I guess it's very difficult to troubleshoot without replicating it.

**Comment by SHPEK1** [comment]
@sum1els Its different issue, that one on X-Plane fb page is about that curson keeps in "spinning knob" mode. This here is different.

**Comment by sum1els** [comment]
@SHPEK1 ah ok ... i have not seen or experienced either cursor issue, so wasn't sure.

**Comment by sum1els** [comment]
@debeljak22 i didn't make this plugin :) .. I know of many people using ZHSI (and obviously ExtPlane plugin), including me, without this issue. I was going to suggest you start with just the ExtPlane plugin and see, but you already done it. The only main difference between the newer version and the old 0.1 you are using, is the addition of a modifier for strings, so I assume when you say ZHSI is not running properly, you mean strings in ZHSI ?

**Comment by debeljak22** [comment]
so sorry @SHPEK1 but thanks for response I ment to send to message to @vranki :) . ZHSI works completely fine with old 0.1 only issue that I have it is very slow and leggy. Thanks

**Comment by Berzerker** [comment]
Seems like this is very old, but I'm experiencing this issue on the latest version of XP11 and ExtPanel, has this been looked into at all?

**Comment by BravoFoxMike** [comment]
I switched to freeware WebFMC, which does not have this issue at all.

**Comment by Berzerker** [comment]
That's great, but I'd be using ExtPanel for Streamdeck integration to send some telnet commands, so that's unfortunately not relevant to my needed fix.

**Comment by waynepiekarski** [comment]
There isn't much that we can look into right now because the problem is very rare and hard to reproduce. Somehow @vranki needs to see this problem on their machine, so they can attach a debugger and see what the problem is. Make sure you have tried the latest version, and also it would be interesting to see if the problem happens with no other plugins running, or a new X-Plane install. It could be another plugin interfering.

**Comment by vranki** [comment]
Yep, still no clue why this happens. ExtPlane does nothing graphical. Could those suffering from the issue send details of which X-Plane version they run, which operating system, and any other details that might help finding out the issue.

**Comment by itchy2** [comment]
Same issue with me. 
X-Plane 11.41r1 on Windows 10. 
"Biggest" Issue on my side: Changing views by holding right mouse button: Sometimes (no clue when exactly) Xplane is not leaving the "change view" mode by leaving the right mouse button. I need to click it again to leave the change view mode.
Nothing helpful in the logs.

**Comment by btbenedi** [comment]
Same issue here
X-Plane 11.41r1 on Windows 10

Sometimes I get the above right mouse move view issue, other times, certain switches just can't hit, like the autofeather on Carenado 1900D for instance, had zero issues before this plugin, first time with the plugin on couldn't click switches, etc. Plugin removed, everything back to normal for mouse. Very odd issue indeed.
